.github/workflows/upload_component.yml

```diff
@@ -12,10 +9,10 @@ jobs:
       - uses: actions/checkout@master
 
       - name: Upload components to component service
-        uses: espressif/github-actions/upload_components@master
+        uses: david-cermak/github-actions/upload_components@feature/upload_component_hooks
```

Comment:
Just don't forget to revert this, before merging

Reply:
:-) Thanks for reminder, I will. Forgot to mark it *Draft* while some WIP commits still present.

Reply:
Done.

Did it as @igrr suggested in the referenced PR. This way it's less code and runs faster (builds the docs just once and uses it for both the `gh-pages` and the component registry), so I think it's okay as we have just one component.